x-pack/plugins/maps/public/connected_components/toolbar_overlay/_toolbar_overlay.scss

```diff
@@ -10,7 +10,7 @@
   position: relative;
   transition: transform $euiAnimSpeedNormal ease-in-out, background $euiAnimSpeedNormal ease-in-out;
 
-  .euiButtonIcon:not(.euiButtonIcon--fill) {
```

Comment:
To avoid this issue happening again in the future, would it be better to apply and style your own custom class name, rather than target the EUI class names? That way if EUI changes the class name again in the future, your styles here won't be affected.

Reply:
Thanks for the feedback. I added mapToolbarOverlay__buttonIcon-empty class when the euiButtonIcon is not filled.